### ISSUE TITLE
Sequence Element Miss Match - All Elements get deleted

### ISSUE THREAD

**Issue by PSepton** [smeighan/xLights#1505]
I am on the latest version.  19-15C 

When I open ANY SEQUENCE it states I have a Sequence Element Miss Match, I can delete or Map Effects, however neither resolve and get the same result where it deletes everything but the timing tracks.  ALL SEQUENCES are the same.  Please see attached.

https://youtu.be/VQBLlf84wKg

Last Version I was on was. 19-09

I am using a MAC Latest build fyi.

**Comment by PSepton** [comment]
I should note, I re-installed it, and of course restarted the software.  Even though this is a mac haha.  Any help would be appreciated.  I am in no rush, I can wait until next release if it seems to unique to me.  I reverted to the previous version I had and am now getting the same error.  I am going to try to restart my Mac... (I can't see that helping but figure I will do it)

**Comment by PSepton** [comment]
Ok I restarted the Mac, and update it too to the latest build.  Still same issue.

**Comment by AzGilrock** [comment]
That youtube link isn't working.  Are you in the right show folder that matches the sequences?  The mismatch happens when the sequence contains models that are not in the current show folder.

**Comment by PSepton** [comment]
I just made it so it could be shared.  I also am seeing now that I am looking no design elements at all. 
![screen shot 2019-03-06 at 8 49 15 pm](https://user-images.githubusercontent.com/48300567/53924244-bbdf1a00-4051-11e9-8376-0b3b432bc0d4.png)
![screen shot 2019-03-06 at 8 48 53 pm](https://user-images.githubusercontent.com/48300567/53924245-bbdf1a00-4051-11e9-8e3c-658b69ad4702.png)

**Comment by PSepton** [comment]
I can also make a better quality video if you needed.  But it is funny because the layers load as if they are correct, I see them when they are saying I need to map.
![screen shot 2019-03-06 at 8 53 50 pm](https://user-images.githubusercontent.com/48300567/53924307-006ab580-4052-11e9-9fc8-62e3430c156b.png)

**Comment by AzGilrock** [comment]
Yeah well you need to solve why you don't have models in your layout screen first.

**Comment by keithsw1111** [comment]
I reckon you changed show folders and thus cant see any of your models. As gil says you need to solve that first.

**Comment by PSepton** [comment]
OK.  What File would be the models be in?  I am still having issues.  I have the right show folder... It is absolutely related to my show.  I went to the imported and no issues with the one I tested.  But my own, not so much.

It worked prior to me upgrading, but not after.  Honestly... I just started re-doing my show, so I will lose a bit of work I guess... if I have to rebuild, but I have last years backed up.  So if i can recover it would make me smile.  

 I should point out... everything else is there but the models.  

Odd eh?


![screen shot 2019-03-07 at 7 32 03 pm](https://user-images.githubusercontent.com/48300567/53996779-28b8e980-4110-11e9-89ec-1925cb45553c.png)
![screen shot 2019-03-07 at 7 24 43 pm](https://user-images.githubusercontent.com/48300567/53996781-28b8e980-4110-11e9-919a-2e07bdf0e303.png)
